_includes/specials/taproot/en/xx-ptlcs.md

```diff
@@ -0,0 +1,152 @@
+easier for block chain surveilance organizations to identify usage
+patterns and, perhaps, make effective guesses about information specific
+to individual users.
+
+### The PTLC solution
+
+In the previous min.sc-style scripts, we had a function that would only
+return true if it was passed a particular value chosen in advance (the
+preimage).  A [signature adaptor][topic adaptor signatures] is similar
+in that it can only be transformed into a valid signature if passed a
+revealed value (the *scalar*).  If we ignore [multisignatures][topic
+multisignature] for the moment, this allows us to transform the HTLC
+scripts from earlier into the following PTLCs:
+
+```hack
+(pk($bob) && pk($alice_adaptor)) || (pk($alice) && older(10))
+(pk($carol) && pk($bob_adaptor)) || (pk($bob) && older(5))
+```
+
+In short, Carol gives Alice the EC *point* for her hidden scalar, Alice
+uses that with a public key she chooses to create a signature adaptor
+that she gives to Bob; Bob can use the same point with a public key
+he chooses to create an adaptor he gives to Carol.  Carol reveals the
+scalar by transforming Bob's adaptor into a valid signature, claiming
+Bob's coins.  Bob recovers the scalar from the valid signature, allowing
+him to transform Alice's adaptor into its own valid signature, claiming
+her coins.
+
+This solves the onchain linkability problem against onchain surveilance because
+all anyone looking at the block chain sees are a bunch of valid
+signatures for distinct public keys.  Third parties can't know that
+adaptors were used, much less what scalar those adaptors were based upon.
+
+However, the procedure above doesn't prevent surveilance nodes who participate
+in the routing from linking together the payments.  If all the payments
+are based on the same scalar, then all the payments are just as linked
+as if they used a hashlock and preimage.  This can be fixed by each
+routing node choosing their own scalar and then removing its
+corresponding point as the payment passes through its node.  Let's
+revise our example:
+
+As before, Carol gives Alice the point for her scalar, but this time
+Alice also requests a point from Bob.  Alice constructs the adaptor she
+gives Bob using the aggregation of both Carol's point and Bob's point.
+Bob knows his point, so he's able to subtract that out from the adaptor
+he receives from Alice.  Using the resultant EC point (which Bob doesn't
+know is now just the point Alice originally received from Carol), Bob
+constructs the adaptor he gives to Carol.  Carol knows the scalar for
+that final point and so converts Bob's adaptor into a valid signature.
+As before, Bob recovers Carol's scalar from her signature and uses it
+and his own scalar to convert Alice's adaptor into a valid signature.
+
+In the two hops in this path, Alice→Bob and Bob→Carol, two different
+EC points and scalars were used, eliminating linkability.  We can apply
+this to the same longer paths we examined when considering HTLC privacy.
```

Comment:
It's not clear to me which longer paths are being referred to here. Also "same longer" seems contradictory.

Reply:
I think it might be unclear to you because you only read the text in the diff.  In the preview, there are two images (one earlier, one immediately after this paragraph) that illustrate a longer path.  I'll change the full stop at the end of this sentence to a colon to maybe help make that a bit clearer.